### PR TITLE
adding package-lock

### PR DESCRIPTION
**Pull request by zhangium**
bump so update to @brightspace-ui-labs/grade-result can get pulled in

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4402,7 +4402,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#a5e36a2617a37084455cd2e9493c9a1fd3c1a873",
+      "version": "github:BrightspaceHypermediaComponents/activities#3efef52176713403610e89b1a4157d32e768821a",
```
Comment (zhangium) [inline package-lock.json:4405]: @spandanabaruah @emil-furniss

File: package-lock.json [hunk 2]
```diff
@@ -4531,7 +4531,7 @@
       }
     },
     "d2l-awards-leaderboard-ui": {
-      "version": "github:Brightspace/awards-leaderboard-ui#e298330c6f4718ada9a1beb9bfe7afa31fb0a131",
+      "version": "github:Brightspace/awards-leaderboard-ui#f10354be714b15d84f81d1f04cd6ad48c19342d4",
```
Comment (zhangium) [inline package-lock.json:4534]: @cbelan-d2l

File: package-lock.json [hunk 3]
```diff
@@ -5314,11 +5314,11 @@
       }
     },
     "d2l-sequences": {
-      "version": "github:BrightspaceHypermediaComponents/sequences#6e2cd3cb8728adb7e9268768df0509fd87a93dd2",
+      "version": "github:BrightspaceHypermediaComponents/sequences#44fdc0c1b92067ef80a445186d9c30aa7309d3ea",
```
Comment (zhangium) [inline package-lock.json:5317]: @imcdonald-d2l

File: package-lock.json (hunk 2 above)
Comment (cbelan-d2l) [reply package-lock.json:4534]: 👍

**Pull request merged**